web/documentserver-example/csharp/DocEditor.aspx

```diff
@@ -290,6 +316,8 @@
             config.events['onRequestReferenceData'] = onRequestReferenceData;
             // prevent switch the document from the viewing into the editing mode for anonymous users
             config.events['onRequestEditRights'] = onRequestEditRights;
+            // add link to reference data source file 
```

Comment:
This comment duplicates the context that is already clear from the code below. I recommend deleting it.

Reply:
All comments that duplicate context have been removed